pkg/virt-handler/cgroup/cgroup_v2_manager.go

```diff
@@ -71,7 +73,17 @@ func (v *v2Manager) Set(r *runc_configs.Resources) error {
 		log.Log.V(5).Infof("cgroupsv2 device allowlist: rule after appending current+new: type: %d permissions: %s allow: %t major: %d minor: %d", rule.Type, rule.Permissions, rule.Allow, rule.Major, rule.Minor)
 	}
 
-	return v.execVirtChroot(&resourcesToSet, map[string]string{"": v.dirPath}, v.isRootless, v.GetCgroupVersion())
+	subsystemPaths := map[string]string{
+		"target": v.dirPath,
+	}
+	if targetDir, parentPath := filepath.Base(v.dirPath), path.Dir(v.dirPath); targetDir == "container" && strings.HasSuffix(parentPath, ".scope") {
+		// This is needed for crun based installations for a brief period of time
+		// crun will eventually stop configuring both cgroups
```

Comment:
Can you please create a tracking issue to eventually remove this and link it here?